HowToAddANewService.md

```diff
@@ -0,0 +1,157 @@
+Tutorial on how to add a badge for a service
+============================================
```

Comment:
Could you move this to `doc/TUTORIAL.md`, and add a link to it in `CONTRIBUTING.md`?